aws/_base/main.tf

```diff
@@ -38,7 +38,12 @@ resource "aws_spot_fleet_request" "runner" {
       }
 
       tags = {
-        Name = "Schutzbot runner"
+        Name            = "Schutzbot runner"
+        Job_name        = var.job_name
+        Project         = var.project
+        Branch          = var.Branch
+        Pipeline_id     = var.pipeline_id
+        Pipeline_source = var.pipeline_source
       }
```

Comment:
Can you also add `Workload` statically set to `CI Runner`, so we can group our workloads simply (I don't want to rely on names).